src/main/scala/org/bitcoins/core/gen/WitnessGenerators.scala

```diff
@@ -120,7 +120,7 @@ trait WitnessGenerators extends BitcoinSLogger {
     val flags = Policy.standardScriptVerifyFlags
     val (creditingTx,outputIndex) = TransactionGenerators.buildCreditingTransaction(witScriptPubKey,amount)
     val (unsignedSpendingTx,inputIndex) = TransactionGenerators.buildSpendingTransaction(creditingTx,EmptyScriptSignature,outputIndex,witness)
-    val unsignedWtxSigComponent = WitnessV0TransactionSignatureComponent(unsignedSpendingTx,inputIndex,witScriptPubKey,flags,amount)
+    val unsignedWtxSigComponent = WitnessV0TransactionSignatureComponent(unsignedSpendingTx,inputIndex,witScriptPubKey,flags,amount, SigVersionWitnessV0)
```

Comment:
Instead of passing the constant `SigVersionWitnessV0` I probably need to use `BitcoinScriptUtil.parseSigVersion`